docs/Merge-MarkdownHelp.md

```diff
@@ -17,35 +17,22 @@ Merge-MarkdownHelp [-Path] <String[]> [-OutputPath] <String> [-Encoding <Encodin
 ```
 
 ## DESCRIPTION
-If we have similar modules, or different version of the same module,
-we are likely to have a lot of duplicated markdown in them.
+Similar modules, or different versions of the same module, often contain duplicate content.
 
-Merge-MarkdownHelp allows you to keep all of them into a single markdown files.
-It uses `applicable:` yaml metadata field to identify what versions or tags are applicable.
-It acts on two levels: for the whole cmlets and for individual parameters.
+Merge-MarkdownHelp merges the multiple markdown files into a single markdown file. It uses the `applicable:` yaml metadata field to identify what versions or tags are applicable. It acts on two levels: for the whole cmdlet and for individual parameters.
```

Comment:
please, prefer one sentence per line in markdown http://rhodesmill.org/brandon/2012/one-sentence-per-line/

gist:
Soft linebreak doesn't result in a new line in markdown.
It makes the reviewer much easier to conduct, because git operates on a line granularity.